src/app/models/usermodel.js

```diff
@@ -134,7 +135,9 @@ window.addEventListener('focus', () => {
             }
         });
     });
-});
+}, 90000);
```

Comment:
is that 90 seconds?  That seems like a pretty long amount of time to wait for accounts, but I'm probably missing something

Reply:
The check happens just in case you went to another tab and logged in. If you leave the tab and return very often, you can rack up a bunch of checks. 90 seconds could be a bit long, but it's an uncommon case, and most of the time you won't have just left, returned, then left and logged in.